### PR TITLE
fix websocket ping variables issues

### PR DESCRIPTION
**Pull request by brooklet**
fixed the problem that the websocket ping_timeout and ping_interval parameter settings did not take effect

### CODE REVIEW

**Comment by ahopkins** [comment]
@brooklet Can you take a look at the failing tests?

File: sanic/server.py [hunk 1]
```diff
@@ -955,14 +955,14 @@ def serve(
 def _build_protocol_kwargs(
     protocol: Type[HttpProtocol], config: Config
 ) -> dict:
-    if hasattr(protocol, "websocket_timeout"):
+    if (dir(protocol).__contains__("websocket_handshake")):
```
Comment (ahopkins) [inline sanic/server.py:958]: Why are you checking for properties this way? Please avoid using `dir()` here.

File: sanic/server.py [hunk 2]
```diff
@@ -955,14 +955,14 @@ def serve(
...
         return {
-            "max_size": config.WEBSOCKET_MAX_SIZE,
-            "max_queue": config.WEBSOCKET_MAX_QUEUE,
-            "read_limit": config.WEBSOCKET_READ_LIMIT,
-            "write_limit": config.WEBSOCKET_WRITE_LIMIT,
-            "ping_timeout": config.WEBSOCKET_PING_TIMEOUT,
-            "ping_interval": config.WEBSOCKET_PING_INTERVAL,
+            "websocket_max_size": config.WEBSOCKET_MAX_SIZE,
+            "websocket_max_queue": config.WEBSOCKET_MAX_QUEUE,
```
Comment (ahopkins) [inline sanic/server.py:961]: Why are these being renamed `websocket_`?

**Comment by ashleysommer** [comment]
@brooklet we want to try to get this merged before the Sanic 2020-09 release. Are you able to make those changes suggested?

File: sanic/server.py (hunk 1 above)
Comment (ahopkins) [reply sanic/server.py]: reverted to `hasattr`, but I might circle back on this to see why we are not using `isinstance` here, which seems more logical. Do we need duck typing for some reason I am unaware of?

File: sanic/server.py (hunk 2 above)
Comment (ahopkins) [reply sanic/server.py:929]: Left this change in, but I updated the tests so that the change was carried through consistently.

**Review by ahopkins — APPROVED** [review]

**Pull request merged**